src/prototype_room_utils.js

```diff
@@ -8,6 +8,15 @@ Room.prototype.sortMyRoomsByLinearDistance = function(target) {
   return _.sortBy(Memory.myRooms, sortByLinearDistance);
 };
 
+Room.prototype.nearestRoomName = function(roomsNames, limit) {
```

Comment:
no because roomsNames come frome needEnergyRooms. i could use it inside but it still allow us to use it with myRooms for example somewhere else  :)